modules/security-context-constraints-rbac.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+[id='role-based-access-to-ssc-{context}']
+= Role-based access to Security Context Constraints
+
+You can specify SCCs as a resource that is handled by RBAC. This allows
+you to scope access to your SCCs to a certain project or to the entire
+cluster. Assigning users, groups, or service accounts directly to an
+SCC retains cluster-wide scope.
+
+To include access to SCCs for your role, specify the `scc` resource
+when creating a role.
+
+----
+oc create role <role-name> --verb=use --resource=scc --resource-name=<scc-name> -n <namespace>
+----
+
+This results in the following role definition:
+
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+...
+  name: role-name <1>
+  namespace: namespace <2> 
+...
+rules:
+- apiGroups:
+  - security.openshift.io <3>
+  resourceNames:
+  - my-permitting-scc <4>
```

Comment:
I know that in the above CLI command you used values in <> to show that you replace, but I think you used the rest of the exact values in the yaml here, like role-name, namespace, etc. Should my-permissing-scc here and <scc-name> in the above command be the same value?

Reply:
You're correct - I adjusted the values and didn't update this entry. They both should now read `scc-name`.